src/server/game/Spells/SpellMgr.cpp

```diff
@@ -7421,6 +7421,11 @@ void SpellMgr::LoadDbcDataCorrections()
         spellInfo->AttributesEx3 |= SPELL_ATTR3_ALWAYS_HIT;
     });
 
+    ApplySpellFix({ 13262, 31252 }, [](SpellEntry* spellInfo)
+    {
+        spellInfo->AttributesEx |= SPELL_ATTR1_ALLOW_WHILE_STEALTHED;
+    });
```

Comment:
No.

just add the spell id's to L#4099 `spellInfo->AttributesEx |= SPELL_ATTR1_ALLOW_WHILE_STEALTHED` already exists there. No need for duplicate entries

Reply:
Done, but we still need to know if these lines are actually needed.

Reply:
Is not part of the PR anymore.